Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1036,3 +1036,7 @@
 #if !defined(ENABLE_QUICKLOOK_SANDBOX_RESTRICTIONS) && PLATFORM(IOS)
 #define ENABLE_QUICKLOOK_SANDBOX_RESTRICTIONS 1
 #endif
+
+#if !defined(ENABLE_NOTIFY_BLOCKING) && PLATFORM(IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED > 170400
```

Comment:
Is the version number correct here?

Reply:
Yes, looking closer, this is intentional.

Thanks for reviewing!